arcgis-ios-sdk-samples/Analysis/Statistical query/README.md

```diff
@@ -1,17 +1,33 @@
 # Statistical query
 
-This sample demonstrates how to query a feature table to get aggregated statistics back for a specific field.
+Query a table to get aggregated statistics back for a specific field.
 
-## How to use the sample
+![Statistical query results](statistical-query.png)
+
+## Use case
 
-Select the appropriate switch to filter features by attributes and/or within the current extent. Click the button to see basic statistics displayed for world cities.
+For example, a county boundaries table with population information can be queried to return aggregated results for total, average, maximum, and minimum population, rather than downloading the values for every county and calculating statistics manually.
+
+## How to use the sample
 
-![](image1.png)
+Pan and zoom to define the extent for the query. Use the 'Cities in current extent' checkbox to control whether the query only includes features in the visible extent. Use the 'Cities grater than 5M' checkbox to filter the results to only those cities with a population greater than 5 million people. Tap 'Get statistics' to perform the query. The query will return population-based statistics from the combined results of all features matching the query criteria.
```

Comment:
```suggestion
Pan and zoom to define the extent for the query. Use the "Cities in current extent" checkbox to control whether the query only includes features in the visible extent. Use the "Cities grater than 5M" checkbox to filter the results to only those cities with a population greater than 5 million people. Tap "Get statistics" to perform the query. The query will return population-based statistics from the combined results of all features matching the query criteria.
```
Probably change all single quote to double when referring to UIControl elements in the sample.